OperationCode library not required for rake tasks

Originally reported in #272. 

After migrating to file based secrets rake tasks are broken.

```
docker-compose run web rake db:create
rake aborted!
NameError: Cannot load `Rails.application.database_configuration`:
uninitialized constant OperationCode
(erb):7:in `<main>'
/usr/local/bundle/gems/railties-4.2.5.2/lib/rails/application/configuration.rb:104:in `dat
abase_configuration'
/usr/local/bundle/gems/activerecord-4.2.5.2/lib/active_record/railtie.rb:41:in `block (3 l
evels) in <class:Railtie>'
/usr/local/bundle/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
NameError: uninitialized constant OperationCode
(erb):7:in `<main>'
/usr/local/bundle/gems/railties-4.2.5.2/lib/rails/application/configuration.rb:104:in `dat
abase_configuration'
/usr/local/bundle/gems/activerecord-4.2.5.2/lib/active_record/railtie.rb:41:in `block (3 l
evels) in <class:Railtie>'
/usr/local/bundle/gems/rake-11.2.2/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:create => db:load_config
(See full trace by running task with --trace)
make: *** [db_create] Error 1
```

--- a/Rakefile
+++ b/Rakefile
@@ -2,5 +2,6 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path('../config/application', __FILE__)
+require File.expand_path('../lib/operation_code', __FILE__)
 
 Rails.application.load_tasks

